lib/get_player_input.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'tic_tac_toe.rb'
+
+def get_player_input(input, player, board=Array.new(3){Array.new(3, "_")})
+    puts "Player #{player}, enter your placeholder in format num1, num2"
+    input = input.gets.chomp
+
+    row = input.split(",")[0].to_i
+    col = input.split(",")[1].to_i
+
+    if board[row][col] == "_"
+        board[row][col] = player
+    else
+        false
+    end
+end
```

Comment:
consider splitting this function into 2, 
1 to get the user input
2 to update the board